pages.pl/netbsd/*: add Polish translations

paulinek13 opened the pull request:
<!--
Thank you for contributing!
Please fill in the following checklist, removing items that do not apply.
See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
-->

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

This PR adds Polish translations for 6 NetBSD-specific commands that were previously untranslated: `cal`, `chpass`, `df`, `pkgin`, `sed`, and `sockstat`.

acuteenvy commented on pages.pl/netbsd/chpass.md:
```diff
@@ -0,0 +1,29 @@
+# chpass
+
+> Dodaj lub zmień informacje w bazie danych użytkownika, w tym powłokę logowania i hasło.
```
```suggestion
> Dodaj lub zmień informacje w bazie danych użytkowników, w tym powłokę logowania i hasło.
```

spageektti reviewed (commented):
Hi. Thank you so much for your contribution!

acuteenvy commented on pages.pl/netbsd/df.md:
```diff
@@ -0,0 +1,32 @@
+
+`df -i`
+
+- Użyj jednostek 1024-bajtowych do zapisu danych przestrzennych:
```
```suggestion
- Użyj jednostek 1024-bajtowych do wyświetlania danych o przestrzeni dyskowej:
```

paulinek13 commented on pages.pl/netbsd/sed.md:
```diff
@@ -0,0 +1,33 @@
+> Zobacz także: `awk`, `ed`.
+> Więcej informacji: <https://man.netbsd.org/sed.1>.
+
+- Zamień wszystkie wystąpienia `apple` (podstawowe wyrażenie regularne) na `mango` (podstawowe wyrażenie regularne) we wszystkich liniach wejściowych i wypisz wynik do `stdout`:
+
+`{{komenda}} | sed 's/apple/mango/g'`
```
Should we keep `apple`and `mango` in their original form as example values in the regular expression, or should we translate `apple` to `jabłko`?

acuteenvy commented on pages.pl/netbsd/sed.md:
```diff
@@ -0,0 +1,33 @@
+
+`{{komenda}} | sed 's/apple/mango/g'`
+
+- Uruchom określony plik (z ang. [f]ile) skryptu i wydrukuj wynik na stdout:
```
```suggestion
- Uruchom określony plik (z ang. [f]ile) skryptu i wydrukuj wynik na `stdout`:
```

acuteenvy commented on pages.pl/netbsd/sockstat.md:
```diff
@@ -0,0 +1,26 @@
+# sockstat
+
+> Wyświetl listę otwartych gniazd internetowych lub UNIX-owych.
+> Uwaga: ten program jest przepisany dla NetBSD 3.0 z programu `sockstat` z FreeBSD.
```
```suggestion
> Uwaga: ten program jest przeróbką programu `sockstat` z FreeBSD dla NetBSD 3.0.
```

acuteenvy reviewed (commented):
Thanks for your contribution.

paulinek13 commented:
Thank you for the feedback! I’ve addressed all the suggested changes and rebased the branch. Please let me know if there’s anything else that needs to be adjusted.

spageektti commented on pages.pl/netbsd/cal.md:
```diff
@@ -0,0 +1,36 @@
+
+`cal -y -j`
+
+- Wyróżnij (z ang. [h]ighlight) dzisiejszą datę i wyświetl [3] miesiące obejmujące tę datę:
```
```suggestion
- Wyróżnij (z ang. [h]ighlight) dzisiejszą datę i wyświetl [3] miesiące ją obejmujące:
```
